pytket/binders/circuit/boxes.cpp

```diff
@@ -515,12 +515,15 @@ void init_boxes(py::module &m) {
       "A box for preparing quantum states using multiplexed-Ry and "
       "multiplexed-Rz gates")
       .def(
-          py::init<const Eigen::VectorXcd &, bool>(),
+          py::init<const Eigen::VectorXcd &, bool, bool>(),
           "Construct from a statevector\n\n"
-          ":param statevector: normalised statevector\n",
+          ":param statevector: normalised statevector\n"
           ":param is_inverse: whether to implement the dagger of the state "
-          "preparation circuit, default to false",
-          py::arg("statevector"), py::arg("is_inverse") = false)
+          "preparation circuit, default to false\n"
+          ":param with_initial_reset: whether to explicitly set the state to "
+          "zero initially (otherwise the initial zero state is assumed)",
```

Comment:
Might be worth adding a further explanation to also cover:
```(by default it is assumed that the initial state is zero and no explicit  reset is applied)```

Reply:
Done.